docs/src/modeler/example_dynamic_data.md

```diff
@@ -0,0 +1,335 @@
+Note that this system does not have an injection device in bus 1 (the reference bus). We can add a source with small impedance directly using a function like:
+```julia
+function add_source_to_ref(sys::System)
+    for g in get_components(StaticInjection, sys)
+        isa(g, ElectricLoad) && continue
+        g.bus.bustype == BusTypes.REF &&
+            error("A device is already attached to the REF bus")
+    end
+
+    slack_bus = [b for b in get_components(Bus, sys) if b.bustype == BusTypes.REF][1]
+    inf_source = Source(
+        name = "InfBus", #name
+        available = true, #availability
+        active_power = 0.0,
+        reactive_power = 0.0,
+        bus = slack_bus, #bus
+        R_th = 0.0, #Rth
+        X_th = 5e-6, #Xth
+    )
+    add_component!(sys, inf_source)
+    return
+end
+
+add_source_to_ref(omib_sys)
+```
+This function attempts to add a infinite source with ``X_{th} = 5\cdot 10^{-6}`` pu if no other device is already attached to the reference bus.
+
+The system can be explored directly using functions like:
+```julia
+collect(get_components(Source, omib_sys))
+collect(get_components(Generators, omib_sys))
+```
+By exploring those it can be seen that the generators are named as: `generator-bus_number-id`. Then, the generator attached at bus 2 is simply named `generator-102-1`.
+
+### Dynamic Injections
+
+We are now interested in attaching to the system the dynamic component that will be modeling our dynamic generator. Later versions will include a parser for `.dyr` files.
+
+Dynamic generator devices are composed by 5 components, namely, `machine`, `shaft`, `avr`, `tg` and `pss`. So we will be adding functions to create all of its components and the generator itself:
+
+```julia
+#Machine
+machine_classic() = BaseMachine(
+    0.0, #R
+    0.2995, #Xd_p
+    0.7087, #eq_p
+)
+
+#Shaft
+shaft_damping() = SingleMass(
+    3.148, #H
+    2.0, #D
+)
+
+#AVR
+avr_none() = AVRFixed(0.0)
+#TG
+tg_none() = TGFixed(1.0) #efficiency
+#PSS
+pss_none() = PSSFixed(0.0)
+
+function dyn_gen_classic(generator)
+    return DynamicGenerator(
+        generator,
+        1.0, #ω_ref
+        machine_classic(), #machine
+        shaft_damping(), #shaft
+        avr_none(), #avr
+        tg_none(), #tg
+        pss_none(), #pss
+    )
+end
+```
+
+The last function receives a static generator, and creates a `DynamicGenerator` based on that specific static generator, with the specific components defined previously. This is a classic machine model without AVR, Turbine Governor and PSS.
+
+Then we can simply create the dynamic generator as:
+
+```julia
+#Collect the static gen in the system
+static_gen = get_component(Generator, omib_sys, "generator-102-1")
+#Creates the dynamic generator
+dyn_gen = dyn_gen_classic(static_gen)
+#Add the dynamic generator the system
+add_component!(omib_sys, dyn_gen)
+```
+
+Then we can simply export our system data to a json file that can be later read as:
+```julia
+to_json(omib_sys, "omib_sys.json")
```

Comment:
I think that our examples using `to_json` should supply a path other the current directory.  We create files in addition to the .json.
```
to_json(omib_sys, "omib/sys.json")
```